bridge/.eslintrc.json

```
@@ -25,13 +25,19 @@
         "prettier" // eslint-config-prettier
       ],
       "plugins": [
+        "no-only-tests", // https://github.com/keptn/keptn/issues/8526: disallow only in commits
         "@typescript-eslint", // @typescript-eslint/eslint-plugin
         "import",
         "eslint-comments", // eslint-plugin-eslint-comments
         "promise", // eslint-plugin-promise
         "prettier" // eslint-plugin-prettier
       ],
       "rules": {
+        "no-only-tests/no-only-tests": [
+          "error", {
```

The pipeline is failing because of prettier. Could you run `prettier:fix` with yarn/npm or adapt the following lines and commit the changes?

```suggestion
          "error",
          {
```

---

fixed, thanks!